core/src/main/scala/org/apache/spark/sql/delta/sources/DeltaSQLConf.scala

```diff
@@ -810,6 +810,15 @@ trait DeltaSQLConfBase {
           |""".stripMargin)
       .booleanConf
       .createWithDefault(false)
+
+  val DELTA_CHECK_REQUIRED_SPARK_CONF =
+    buildConf("checkRequiredSparkConf.enabled")
```

Comment:
nit: `requiredSparkConfsCheck.enabled` to follow existing config key style such as `alterLocation.bypassSchemaCheck` and `retentionDurationCheck.enabled`.
`DELTA_CHECK_REQUIRED_SPARK_CONF` -> `DELTA_REQUIRED_SPARK_CONFS_CHECK`

Reply:
done